sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -262,16 +265,17 @@ def _get_online_features_dict_remotely(
         request["features"] = features
     else:
         request["feature_service"] = features.name
-    for _ in range(25):
+    for attempt in range(25):
         # Send the request to the remote feature server and get the response in JSON format
         response = requests.post(
             f"{endpoint}/get-online-features", json=request, timeout=30
         ).json()
+        _logger.info(f"Attempt: {attempt}, {response}")
```

Comment:
Should there be more detail here? Will our test logs just have a bunch of `attempt` statements without context?

Reply:
This was for debugging in the github action; I'm going to remove this.